symfony2/yaml/forms.yml

```diff
@@ -53,4 +53,58 @@ questions:
             - {value: "validation_groups", correct: true}
             - {value: "groups_validation", correct: false}
             - {value: "validator_groups",  correct: false}
-            - {value: "groups_validator",  correct: false}
\ No newline at end of file
+            - {value: "groups_validator",  correct: false}
+    -
+        question: 'Which form event exist?'
+        answers:
+            - {value: "FormEvents::PRE_SET_DATA",   correct: true}
+            - {value: "FormEvents::SUBMIT",         correct: true}
+            - {value: "FormEvents::POST_SET_DATA",  correct: true}
+            - {value: "FormEvents::POST_REQUEST",   correct: false}
+    -
+        question: 'What data is inside FormEvent object at FormEvents::PRE_SET_DATA?'
+        answers:
+            - {value: "Model data",             correct: true}
+            - {value: "Model data normalized",  correct: false}
+            - {value: "Request data",           correct: false}
+            - {value: "View data",              correct: false}
+    -
+        question: 'What data is inside FormEvent object at FormEvents::PRE_SUBMIT?'
+        answers:
+            - {value: "Model data",             correct: false}
+            - {value: "Model data normalized",  correct: false}
+            - {value: "Request data",           correct: true}
+            - {value: "View data",              correct: false}
+    -
+        question: 'Which text type exist?'
```

Comment:
Better to have something like 'Which ones of these types extends from TextType ?' no ?